### PR TITLE
Automation: Deletion of component from inspector, right side panel & via keyboard actions

### PR DESCRIPTION
**Pull request by Nandinisaha13**
fixes: #8345

### CODE REVIEW

File: cypress-tests/cypress/e2e/editor/inspectorHappypath.cy.js [hunk 1]
```diff
@@ -170,4 +170,10 @@ describe("Editor- Inspector", () => {
     cy.notVisible(commonWidgetSelector.draggableWidget("button1"));
     cy.apiDeleteApp();
   });
+  it("should verify deletion of component from inspector", () => {
+    cy.dragAndDropWidget("button", 500, 500);
+    cy.get(commonWidgetSelector.sidebarinspector).click();
+    deleteComponentFromInspector("button1");
+
```
Comment (emidhun) [inline cypress-tests/cypress/e2e/editor/inspectorHappypath.cy.js:177]: - I think we need to add assertion to verify the deletion of component and the toast message, after the deletion.
- Let's create one more version and verify that the components is not exist on canvas.

File: cypress-tests/cypress/support/utils/inspector.js [hunk 2]
```diff
@@ -32,3 +32,8 @@ export const verifyValue = (node, type, children, index = 0) => {
     .eq(index)
     .verifyVisibleElement("have.text", type);
 };
+export const deleteComponentFromInspector = (node) => {
+  cy.get('[data-cy="inspector-node-components"] > .node-key').click();
+  cy.get(`[data-cy="inspector-node-${node}"] > .node-key`).click();
+  cy.get(cy.get('[style="height: 13px; width: 13px;"] > img').click());
```
Comment (emidhun) [inline cypress-tests/cypress/support/utils/inspector.js:38]: I think we can only use `cy.get('[style="height: 13px; width: 13px;"] > img').click()` here.

**Review by emidhun — CHANGES_REQUESTED** [review]
@Nandinisaha13 I have added few changes, Please add the cases and let me know if you need any help.

**Comment by Nandinisaha13** [comment]
@emidhun please take a look have fixed the comments.

**Comment by emidhun** [comment]
@Nandinisaha13  Could you please add the delete functionality from the right side panel as well?
I think Only this case is pending on deletion.
Please let me know if it is possible.
Rest everything is looking good.

**Review by emidhun — APPROVED** [review]
LGTM

**Pull request merged**